src/main/scala/org/dia/algorithms/mcc/MCCOps.scala

```diff
@@ -216,31 +251,42 @@ object MCCOps {
       edgeList: RDD[(Int, Iterable[MCCEdge])], iteration: Int,
       buckerSize: Int,
       minGraphLength: Int,
-      sc: SparkContext): Array[(Int, Iterable[MCCEdge])] = {
+      sc: SparkContext): Array[(Int, (Iterable[MCCEdge], mutable.MutableList[mutable.HashSet[String]]))] = {
```

Comment:
Would that work given the logic of the code?
I'm fine with returning the array, it's just the tuple type inside of the array.
If the map step returns Array[(Int, Iterable[MCCEdge]) then it should be fine I think.